src/styleguide/_patterns/05-pages/00-homepage.json

```diff
@@ -0,0 +1,53 @@
+{
+  "title": "Buildit @ Wipro Digital",
+  "control": {
+    "label": "Menu"
+  },
+  "hero": {
+    "background": "/images/buildit-roundtable-bw.jpg",
+    "content": [
+      {
+        "pullout": "Buildit is a transformation studio.<br>We are a company that helps create better companies."
+      },
+      {
+        "paragraph": "<a href='link.pages-about' class=\"grav-c-cta grav-c-cta--inverted\">About us</a>"
+      }
+    ]
+  },
+  "excerpt": {
+    "medium": "We solve hard engineering problems withour costumers,<br>and help them deliver world-class products."
```

Comment:
Small typo, should be 'with our'

Reply:
... and `costumer` should be `customer` 😄